signing/paths/sign.yaml

```diff
@@ -49,6 +49,8 @@ post:
           BLOCK_V2 (DENEB):
             value:
               type: "BLOCK_V2"
+              proofs:
+                - relayer_signature: "0x1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505cc411d61252fb6cb3fa0017b679f8bb2305b26a285fa2737f175668d0dff91cc1b66ac1fb663c9bc59509846d6ec05345bd908eda73e670af888da41af171505"
```

Comment:
This change is additive but I am not sure if it deserves a new version. If we don't bump the version, the field has to be optional, and may not be present even if the web3signer requests it on the public_keys return

Reply:
agree, optional, or new interface.